### ISSUE TITLE
How to train on Waymo dataset ？

### ISSUE THREAD

**Issue by wbzheng11** [Rock-100/MonoDet#17]
The code only provides the preparation method of the KITTI datasets. 

Can the author introduce how to prepare the training set and validation set data for the Waymo datasets?

Thank you ！
